ramls/patron-block-condition.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Patron block conditions schema",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "A UUID identifying the condition",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    },
+    "name": {
+      "type": "string",
+      "description": "The name of the condition"
+    },
+    "blockBorrowing": {
+      "type": "boolean",
+      "description": "This flag indicates the block for borrowing"
+    },
+    "blockRenewals": {
+      "type": "boolean",
+      "description": "This flag indicates the block for renewals"
+    },
+    "blockRequests": {
+      "type": "boolean",
+      "description": "This flag indicates the block for requests"
+    },
+    "valueType": {
+      "type": "string",
+      "description": "Defines limit type",
+      "enum": [
+        "Integer",
+        "Double"
+      ]
+    },
+    "message ": {
+      "type": "string",
+      "description": "A message to be displayed"
+    },
+    "metadata": {
+      "type": "object",
+      "description": "Metadata about creation and changes to patron block conditions, provided by the server (client should not provide)",
+      "$ref": "raml-util/schemas/metadata.schema"
+    }
+  },
+  "required": [
+    "id",
+    "name"
+  ]
```

Comment:
Several properties are optional. Can you extend the description of each optional property explaining the default value that is assumed when the property is left out?

Reply:
updated